Source/relay/Guide/Items of the Month/Bird-a-Day Calendar.ash

```diff
@@ -42,6 +42,12 @@ void IOTMBirdADayCalendar(ChecklistEntry [int] resource_entries)
 
                 description.listAppend(HTMLGenerateSpanOfClass(name, "r_bold") + modValue);
             }
+        } else {
+            main_title = "Discover your daily Bird";
+            subtitle = "";
+            description.listAppend("Use your calendar to get a new skill for the day");
```

Comment:
I like this! I think we should add this reminder to the `Optional Tasks` section like we do for "Modify your lightsaber".